### PR TITLE
Fix pep8/flake8 issues

### PR DESCRIPTION
**Pull request by nnDarshan**
tendrl-bug-id: Tendrl/gluster-integration#330
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @shtripat Please review

File: tendrl/gluster_integration/objects/volume/atoms/named_volume_not_exists/__init__.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 class NamedVolumeNotExists(objects.BaseAtom):
     obj = Volume
```
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/named_volume_not_exists/__init__.py]: not required

File: tendrl/gluster_integration/sds_sync/brick_utilization.py [hunk 2]
```diff
@@ -134,24 +134,43 @@ def _get_thin_pool_stat(device):
 
 def brick_utilization(path):
     """{
+
          'used_percent': 0.6338674168297445,
+
```
Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/brick_utilization.py]: line gap is not required at each line I feel

Comment (nnDarshan) [inline tendrl/gluster_integration/sds_sync/brick_utilization.py]: it complains without the line gap

File: tendrl/gluster_integration/objects/volume/atoms/named_volume_not_exists/__init__.py (hunk 1 above)
Comment (nnDarshan) [inline tendrl/gluster_integration/objects/volume/atoms/named_volume_not_exists/__init__.py]: ack

**Comment by nnDarshan** [comment]
@r0h4n Please review

**Pull request merged**